cast/src/print_utils.rs

```diff
@@ -0,0 +1,507 @@
+use ethers_core::types::{Block, Transaction, *};
+
+pub fn get_pretty_tx_attr(transaction: Transaction, attr: String) -> Option<String> {
+    return match attr.as_str() {
+        "blockHash" => Some(transaction.block_hash.pretty()),
+        "blockNumber" => Some(transaction.block_number.pretty()),
+        "from" => Some(transaction.from.pretty()),
+        "gas" => Some(transaction.gas.pretty()),
+        "gasPrice" => Some(transaction.gas_price.pretty()),
+        "hash" => Some(transaction.hash.pretty()),
+        "input" => Some(transaction.input.pretty()),
+        "nonce" => Some(transaction.nonce.pretty()),
+        "r" => Some(transaction.r.pretty()),
+        "s" => Some(transaction.s.pretty()),
+        "to" => Some(transaction.to.pretty()),
+        "transactionIndex" => Some(transaction.transaction_index.pretty()),
+        "v" => Some(transaction.v.pretty()),
+        "block_hash" => Some(transaction.block_hash.pretty()),
+        "block_number" => Some(transaction.block_number.pretty()),
+        "gas_price" => Some(transaction.gas_price.pretty()),
+        "transaction_index" => Some(transaction.transaction_index.pretty()),
+        "value" => Some(transaction.value.pretty()),
+        _ => None,
+    };
+}
+
+pub fn get_pretty_block_attr<TX>(block: Block<TX>, attr: String) -> Option<String> {
+    return match attr.as_str() {
+        "baseFeePerGas" => Some(block.base_fee_per_gas.pretty()),
+        "difficulty" => Some(block.difficulty.pretty()),
+        "extraData" => Some(block.extra_data.pretty()),
+        "gasLimit" => Some(block.gas_limit.pretty()),
```

Comment:
```suggestion
        "gasLimit" | "gas_limit" => Some(block.gas_limit.pretty()),
```

applies to all fields ex `gasUsed | gas_used`

Reply:
also addressed!